csvcubed/csvcubed/cli/entrypoint.py

```diff
@@ -109,3 +109,4 @@ def inspect_command(log_level: str, csvw_metadata_json_path: Path) -> None:
         inspect(csvw_metadata_json_path)
     except Exception as e:
         log_exception(_logger, e)
+        raise
```

Comment:
I believe we've got something more suitable merged into main now...

https://github.com/GSS-Cogs/csvcubed/blob/main/csvcubed/csvcubed/cli/entrypoint.py#L115